services/.env.example

```diff
@@ -323,6 +327,20 @@ ONAFRIQ_UNIQUE_KEY=
 ONAFRIQ_API_URL=
 ONAFRIQ_CURRENCY_CODE=
 ONAFRIQ_COUNTRY_CODE=
+ONAFRIQ_SFTP_HOST=
+ONAFRIQ_SFTP_PORT=
+ONAFRIQ_SFTP_USERNAME=
+ONAFRIQ_SFTP_PASSPHRASE=
+# Usage: Leave 'empty' in local env, upload the certificate content on remote env.
+ONAFRIQ_SFTP_CERTIFICATE_CONTENT=
+# Usage: Remove '.example' in local development (and fill .pub with right content). Leave empty on remote env, use ONAFRIQ_SFTP_CERTIFICATE_CONTENT instead.
+ONAFRIQ_SFTP_CERTIFICATE_PATH=cert/REDCROSSDRCsftpp.pub.example
```

Comment:
Why is this not cert/REDCROSSDRCsftpp.pub? Is there a scenario where you would use the .example?